react/features/base/tracks/actions.web.ts

```diff
@@ -44,6 +46,9 @@ export function toggleScreensharing(
     return (dispatch: IStore['dispatch'], getState: IStore['getState']) => {
         // check for A/V Moderation when trying to start screen sharing
         if ((enabled || enabled === undefined) && shouldShowModeratedNotification(MEDIA_TYPE.VIDEO, getState())) {
+            if (!isModerationNotificationDisplayed(MEDIA_TYPE.SCREENSHARE, getState)) {
```

Comment:
Why check, since it's sticky and has a defined UID re-showing it won't do anything, will it?